packages/wonder-blocks-grid/components/row.test.js

```diff
@@ -13,11 +13,19 @@ import {mount, unmountAll} from "../../../utils/testing/mount.js";
 describe("Row", () => {
     beforeEach(() => {
         unmountAll();
+
+        // eslint-disable-next-line no-console
+        if (console.error.mockRestore) {
+            // eslint-disable-next-line no-console
+            console.error.mockRestore();
```

Comment:
Could combine to `console.error.mockRestore && console.error.mockRestore()` and just have a single lint disable comment.